docs/runtests.md

```diff
@@ -0,0 +1,51 @@
+# Running Tests
+
+This document demonstrates how to run Cromshell's tests. All tests can be run from the 
+Cromshell directory. 
+
+### Linting files
+
+As a reminder, linter is used to scan your code for styling corrections and issues that 
+may lead to bugs. The main method linting can be executed is through Tox, which will
+run all the linting tools. However, It's possible to run the linting tools individually
+as seen below
+
+    # run all linting commands
```

Comment:
Is there any recommended order for running these individual linting commands?

Reply:
No order needed